eventhub_client/samples/receive/receive.c

```diff
@@ -127,7 +127,7 @@ int Receive_Sample(void)
             // enable connection tracing
             EventHubReceiver_SetConnectionTracing(eventHubReceiveHandle, true);
             // register callback functions to read event payload from the hub
-            errorCode = EventHubReceiver_ReceiveFromStartTimestampWithTimeoutAsync(eventHubReceiveHandle, OnReceiveCB, NULL, OnErrorCB, NULL, nowTimestamp, 1000);
+            errorCode = EventHubReceiver_ReceiveFromStartTimestampWithTimeoutAsync(eventHubReceiveHandle, OnReceiveCB, NULL, OnErrorCB, NULL, nowTimestamp, 10000);
```

Comment:
:-)

---
In reply to: [109302390](https://github.com/Azure/azure-event-hubs-c/pull/11#discussion_r109302390) [](ancestors = 109302390)